### DISCUSSION TITLE
Batch Processing

### DISCUSSION THREAD

**Discussion by ameen-roayan** [Mikubill/sd-webui-controlnet#111] [General]
I am not sure if this is possible or not, but i tried doing batch processing in img2img tab and it will not go past the first frame despite several attempts. Is there something a miss or is it not capable of doing that yet ?

**Comment by ClashSAN** [comment]
In settings tab, tick the "Do not append map to output" checkbox.

**Reply by ljleb** [reply]
As of 4dab9f4790d7bae517ed3269fa98593f26708add this is not necessary anymore. You can adjust the setting to your taste now.

**Comment by AugmentedRealityCat** [comment]
You can also use the XYZ Plot script to animate different ControlNet parameters. Like this example of Shrek transforming into Gigachad: 

That was made by animating the ControlNet Weight parameter + some after effects editing.

https://user-images.githubusercontent.com/114955776/219851074-d99b3fb5-b238-4a73-b6dd-1aea66113a22.mp4
